### DISCUSSION TITLE
Where is the battery shown?

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#99] [Q&A] [answered]
I want to add the battery show in my statusbar. It would be really nice if you could help in that.

**Comment by jhenrique04** [comment]
Hello, so, no bb stats yet, the dev is working on it.

However, if you use a laptop, like me, just put this command in the terminal:

upower -i $(upower -e | grep BAT) | grep --color=never -E "state|to\ full|to\ empty|percentage"

Thats the way i found.

**Comment by prasanthrangan** [comment, marked as answer]
Try adding `battery` in `~/.config/waybar/config.ctl` at the position you want to display it.
